### PR TITLE
feat(view-pager-android): add TabViewPager with switchable paging

### PR DESCRIPTION
**Pull request by ADjenkov**
Create `TabViewPager` layout that extends `ViewPager` and supports disabling swipe gestures over content area. 

`setSwipePageEnabled` - enables/disables touch interactions and overrides the lateral motion transition between views when disabled. _[See Android design guidelines for bottom navigation](https://material.io/guidelines/components/bottom-navigation.html#bottom-navigation-behavior)_

**_NOTE: `TabViewPager` is required since the `TabView` control for android in tns-core-modules should support top/bottom tab items positioning._**

### CODE REVIEW

File: android/widgets/src/main/java/org/nativescript/widgets/TabViewPager.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+import android.content.Context;
+import android.support.v4.view.ViewPager;
+import android.util.AttributeSet;
+import android.view.View;
+import android.view.MotionEvent;
+import android.view.KeyEvent;
+
+// See this thread for more information https://stackoverflow.com/questions/9650265
+public class TabViewPager extends ViewPager {
+    private boolean swipePageEnabled = true;
```
Comment (manoldonev) [inline android/widgets/src/main/java/org/nativescript/widgets/TabViewPager.java:15]: No need to initialize it here as it is set in the constructor.

**Review by manoldonev — APPROVED** [review]

**Comment by ADjenkov** [comment]
run ci

**Comment by SvetoslavTsenov** [comment]
widgets

**Comment by SvetoslavTsenov** [comment]
widgets

**Comment by SvetoslavTsenov** [comment]
widgets

**Comment by SvetoslavTsenov** [comment]
widgets

**Comment by SvetoslavTsenov** [comment]
run ci

**Pull request merged**